src/components/ChannelList/ChannelList.tsx

```diff
@@ -350,8 +351,8 @@ const UnMemoizedChannelList = <
             ) : (
               <Paginator
                 hasNextPage={hasNextPage}
+                isLoading={channelsQueryState.queryInProgress === 'load-more'}
```

Comment:
Would it make sense to "enumerize" stuff like `load-more`? Not sure how many other options are available.

Reply:
@arnautov-anton only two loading states are possible - initial load and `load-more` (pagination). I believe enumeration is not necessary.